src/controller/src/command_executor.cpp

```diff
@@ -31,6 +31,7 @@ void CommandExecutor::run(const size_t commandId, const proto::CommandMessage &m
         BOOST_LOG_TRIVIAL(warning) << "Command " << commandId << " was already running, running parallel.";
     }
     item.status = CommandStatus::STARTING;
+    commands.getStatusRepository()->updateItem(commandId, repositories::CommandStatusRepository::CommandStatusItem::STARTING);
```

Comment:
`commands.getStatusRepository()->updateItem(commandId, item.status);` is korter en misschien netter?